components/higher-order/with-a11y-messages/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { debounce } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from 'element';
+
+/**
+ * A Higher Order Component used to be provide a unique instance ID by component
+ *
+ * @param {WPElement}  WrappedComponent  The wrapped component
+ *
+ * @return {Component}                   Component with an instanceId prop.
+ */
+function withA11yMessages( WrappedComponent ) {
+	return class extends Component {
+		constructor() {
+			super( ...arguments );
+			this.debouncedSpeak = debounce( this.speak.bind( this ), 500 );
+		}
+
+		speak( message, type = 'assertive' ) {
```

Comment:
https://github.com/WordPress/gutenberg/pull/2107#issuecomment-319174903 notwithstanding, `packages/a11y` the default type was chosen as `'polite'`

https://github.com/WordPress/packages/blob/2eb0aac/packages/a11y/src/index.js#L31

Inconsistency could be confusing